### DISCUSSION TITLE
notebook/Math/Probability/Error/

### DISCUSSION THREAD

**Discussion by giscus** [Slowist-Lee/notebook#2] [Announcements]
# notebook/Math/Probability/Error/

https://slowist-lee.github.io/notebook/Math/Probability/Error/

**Comment by Slowist-Lee** [comment]
目前知道的错误：
- 勘误第一题【1/2】处算错了

**Reply by Slowist-Lee** [reply]
- 2022-2023 大题第五题的拒绝域好像写反了

**Comment by LH-0124** [comment]
我用的是更早一版的学解，所以可能部分错误已经更新。
21-22第二学期，第4题最后一空，自由度为n的卡方分布均值应该依概率收敛到n
大题四（4）第一项应为$\frac{9\sigma}{n}$

**Comment by CSGISBME** [comment]
修订： 2021-2022 第二学期 大题三（2）
$P(Y = 0)=\dfrac{2}{3} ，\ P(Y = 1)=\dfrac{1}{3}$ 
$P(M = 0)=(\dfrac{2}{3})^5$

对于 $Z_i = X_iY$:

当 $Y = 0 , Z_i = 0 $

当 $Y = 1$, $Z_i = X_, \ X_i \sim E(1)$

设 $F_{Z_i}(z)$  是 $z$的分布函数，
$F_{Z_i}(z)=P(Z_i \leq z)=P(Z_i = 0)+P(Z_i \leq 1|Y = 1)P(Y = 1)=\dfrac{2}{3}+\dfrac{1}{3}(1 - e^{-z})$
$\text{when}  \ x\geq 0$

$F_M(m)=P(M \leq m)=P(max Z_1,Z_2...Z_5 \leq m)=\dfrac{2}{3}+\dfrac{1}{3}(1 - e^{-m})^5$

综上，分布函数是：

$$
F_M(m)=
\begin{cases}
0 & \text{if } m < 0 \\
\dfrac{2}{3}+\dfrac{1}{3}(1 - e^{-m})^5 & \text{if } m \geq 0
\end{cases}
$$

**Comment by kiri603** [comment]
2020-2021学年
第五大题x的范围应该是x＞=Θ和x＜Θ，试卷上是0
